Frontend/ui-driver/index.js

```diff
@@ -154,6 +157,16 @@ window.onMerchantEvent = function (_event, payload) {
     window.__payload.sdkVersion = "2.0.1"
     console.warn("Process called");
     const parsedPayload = JSON.parse(payload);
+    if (parsedPayload.payload && parsedPayload.payload.hasOwnProperty('onCreateTimeStamp') && parsedPayload.payload.hasOwnProperty('initiateTimeStamp'))
+      {
+        const onCreateTimeStamp = parsedPayload.payload.onCreateTimeStamp;
+        const initiateTimeStamp = parsedPayload.payload.initiateTimeStamp;
+        window.jsonObject["onCreateToBundle"] = prevTimeStamp - onCreateTimeStamp;
+        window.jsonObject["bundleToInitiate"] = initiateTimeStamp - window.prevTimeStamp;
```

Comment:
`bundleToInitiate` has to be handled in js. `initiateTimeStamp - window.prevTimeStamp` this will give correct timings.